src/Gzero/Core/Parsers/NumericParser.php

```diff
@@ -95,8 +97,8 @@ public function parse(Request $request)
 
             // do not reorder this
             if (substr($value, 0, 1) === '!') {
-                $this->operation = '!=';
-                $this->value     = substr($value, 1);
+                $this->operation = '!';
```

Comment:
Why you change operation, now it's not matching $allowedOperations in Query\Condition - this should be a valid database operator